satpy/readers/viirs_active_fires.py

```diff
@@ -0,0 +1,68 @@
+from satpy.readers.netcdf_utils import NetCDF4FileHandler
```

Comment:
This module and all python modules you've created should include the header like the rest of satpy. The copyright author should be "SatPy developers"...I think (I keep changing my mind, but I think this is what I'm going with now). Make sure blank lines still have a `#`. The module should have a docstring `"""` too.